core/src/main/kotlin/supplier/CacheEntitySupplier.kt

```diff
@@ -388,11 +388,14 @@ public class CacheEntitySupplier(private val kord: Kord) : EntitySupplier {
 
     override fun getGuildApplicationCommands(
         applicationId: Snowflake,
-        guildId: Snowflake
+        guildId: Snowflake,
+        withLocalizations: Boolean?
     ): Flow<GuildApplicationCommand> = cache.query<ApplicationCommandData> {
         idEq(ApplicationCommandData::guildId, guildId)
         idEq(ApplicationCommandData::applicationId, applicationId)
-    }.asFlow().map { GuildApplicationCommand(it, kord.rest.interaction) }
+    }.asFlow()
+        .filter { (it.nameLocalizations is Optional.Missing) == withLocalizations }
```

Comment:
```suggestion
        .map {
            when (withLocalizations) {
                true, null -> it
                false -> it.copy(
                    nameLocalizations = Optional.Missing(),
                    descriptionLocalizations = Optional.Missing(),
                )
            }
        }
```
don't filter, just exclude localizations (but only when explicitly specified, not when `withLocalizations == null`)

Reply:
we have the discordBoolean extension function, I'll see if I can use it instead